tests/cis_tests/artificial_norm_script.py

```diff
@@ -34,7 +34,7 @@
 peakThreshold = 0.05
 offsetConvergenceLimit = 0.1
 isLite = True
-Config._config["cis_mode"] = False
+Config._config["cis_mode"]["enabled"] = False
```

Comment:
cis tests, unlike the actual backend above, are allowed to modify config like this because they are just a proof of concept, not officially supported backend.